py-polars/tests/unit/operations/test_group_by_dynamic.py

```diff
@@ -476,17 +476,7 @@ def test_group_by_dynamic_lazy(every: str | timedelta, tzinfo: ZoneInfo | None)
     ]
 
 
-@pytest.mark.parametrize(
-    ("every", "match"),
-    [
-        ("-1i", r"'every' argument must be positive"),
-        (
-            "2h",
-            r"you cannot combine time durations like '2h' with integer durations like '3i'",
-        ),
-    ],
-)
```

Comment:
these two errors are completely different, I've moved the second one out into a separate test below